modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/upgrade/registry/DLServiceUpgradeStepRegistrator.java

```diff
@@ -120,11 +124,20 @@ protected String[] getModuleTableNames() {
 		registry.register(
 			"3.2.3", "3.2.4",
 			new UpgradeDLSizeLimitConfiguration(_configurationAdmin));
+
+		registry.register(
+			"3.2.4", "3.2.5",
+			new DLFileEntryTypePermissionUpgradeConfiguration(
+				_companyLocalService, _resourcePermissionLocalService,
+				_roleLocalService));
```

Comment:
I don't think we should do an upgrade here. Maybe they have it unset on purpose, so we'd be screwing up their preferred permissions.